# Transcriptional profiles analysis of children infected with COVID-19 and malaria infection reveals potential biomarkers

## 1.	Background

**What is the current problem?**

The clinical diagnosis and distinction between malaria and COVID-19 is a challenge to clinicians due to their overlapping symptoms. This causes a potential risk of misdiagnosis and in turn inappropriate treatment therapy provision or death.

**What tools are currently used to distinguish the two?**

* Use of clinical indicators, such as bivariate and multivariate regression analyses.
* Triage classification and grouping of severe infections based on Point of care rapid diagnostic test (RDT) results.
* Implementation of available clinical guidelines on case-by-case management.
* Risk assessment biomarker.
* Clinical-based algorithms. 

**What are we proposing**

We use public datasets of RNA-seq data to quantify RNA transcripts of blood from patients with malaria and COVID-19 infection. 

**To achieve this we ought to:**
1. Use of Next generation sequencing data which provide a more sensitive, specific and accurate method for the etiologic distinction of malaria and COVID-19.
2. Use differential gene expression analysis profiles to identify unique signatures between children patients with malaria and COVID-19. 
3. Determine the common genes differentially expressed between malaria and COVID-19 diseases.

**Targets | Data Collection**

![image](https://github.com/omicscodeathon/RNA-seq-Malaria/blob/main/Images/Datasets%20collection.png)


##### 

## 2. Tools | Workflow of the project 
![image](https://github.com/omicscodeathon/RNA-seq-Malaria/blob/main/Images/workflow.png)

 **Source of datasets**
  - [GEO ](https://www.ncbi.nlm.nih.gov/geo/): public data repository for storing high throughput gene expression data and hybridization arrays, chips, microarrays
  - [Array-express](https://www.ebi.ac.uk/arrayexpress/):  public repository for microarray-based gene expression data
  - [ENCODE](https://www.encodeproject.org/): public research consortium that contains a list of functional elements in the human genome
  - [SRA](https://www.ncbi.nlm.nih.gov/sra) : contains raw sequence data for an experiment
  - [ENA](https://www.ebi.ac.uk/ena/browser/home) : European Nucleotide Archive, for storing raw sequencing data
  - [DDBJ-DRA](https://www.ddbj.nig.ac.jp/dra/index-e.html): stores raw sequences generated by Next Generation Sequencing (NGS)   


## 5.Download the RNA-seq data

 Download the RNA-seq data multiple link using `wget` command
 
Create a `.txt` file, contain all link you want to download :

`ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/069/SRR14853569/SRR14853569_1.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/069/SRR14853569/SRR14853569_2.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/073/SRR14853573/SRR14853573_1.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/073/SRR14853573/SRR14853573_2.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/076/SRR14853576/SRR14853576_1.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/076/SRR14853576/SRR14853576_2.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/080/SRR14853580/SRR14853580_1.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/080/SRR14853580/SRR14853580_2.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/085/SRR14853585/SRR14853585_1.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/085/SRR14853585/SRR14853585_2.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/089/SRR14853589/SRR14853589_1.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/089/SRR14853589/SRR14853589_2.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/090/SRR14853590/SRR14853590_1.fastq.gz
ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR148/090/SRR14853590/SRR14853590_2.fastq.gz`

Save as `children_covid.txt`

Run the command : ` sudo wget -c -b -i children_covid.txt`


**Download ref. genome**

 `sudo wget http://ftp.ensembl.org/pub/release-104/fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna.alt.fa.gz
`

 `sudo wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/GRCh38.primary_assembly.genome.fa.gz`
 

**Ensembl with gene_id is good for featurecount**

 `sudo wget http://ftp.ensembl.org/pub/release-104/gtf/homo_sapiens/Homo_sapiens.GRCh38.104.gtf.gz
`



## 5.Tools used

**Install subread**

Download the subread file:
 
 `sudo wget https://sourceforge.net/projects/subread/files/subread-2.0.1/subread-2.0.1-Linux-x86_64.tar.gz`
 
 Unzip the subread file:
 
 `sudo tar zxvf subread-2.0.1-Linux-x86_64.tar.gz`

Check if all packages are included :

`/subread-2.0.1-Linux-x86_64$ cd bin/`

**Note**: This package contains an aligner and feature counter. (subread-align - short read aligner and featureCounts for feature counter)
exactSNP 
featureCounts  
subindel  
subjunc  
sublong  
subread-align  
subread-buildindex  
utilities


Link the executables into the bin/ folder

`/subread-2.0.1-Linux-x86_64$ cd bin/`

**Build the index** 

`$ cd /my_shared_data_folder/3.ref_genome/index`

` sudo ../../subread-2.0.1-Linux-x86_64/bin/subread-buildindex -o GRCh38_indexed ../GRCh38.primary_assembly.genome.fa > /dev/null 2>&1 &
`

$ cd /my_shared_data_folder/3.ref_genome/index
 sudo ../../subread-2.0.1-Linux-x86_64/bin/subread-buildindex -o homo_sapiens_indexed ../Homo_sapiens.GRCh38.dna.toplevel.fa


**Mapping**

`$ cd /my_shared_data_folder/5.alignment
sudo ../subread-2.0.1-Linux-x86_64/bin/subread-align -t 0 -T 16 -i ../3.ref_genome/index/GRCh38_indexed -r ../4.trimming/cont1_1_paired.fastq.gz -R ../4.trimming/cont1_2_paired.fastq.gz -o cont1.bam > /dev/null 2>&1 &
`

**a for loop for mapping**

`cd /home/user/my_shared_data_folder/4.trimming
for i in `ls -1 *_1_paired.fastq.gz | sed 's/\_1_paired.fastq.gz//'`; do echo ${i};
sudo ../subread-2.0.1-Linux-x86_64/bin/subread-align -t 0 -T 16 -i ../3.ref_genome/index/GRCh38_indexed -r ${i}_1_paired.fastq.gz -R ${i}_2_paired.fastq.gz -o ../5.alignment/${i}.bam; echo "${i} is done"; done > /dev/null 2>&1 &`






## 4. Team Members
* [Nzungize Lambert](https://github.com/nzungizelab), Centre for Research in Infectious Diseases (CRID/LSTM), nzulapa@outlook.com

* [Arnaud Kengne-Ouafo](https://github.com/JAKO-waccbip), Centre for Research in infectious diseases (CRID), yaound√©, Cameroon, jakengne@gmail.com

* [Rissy Makhokha](https://github.com/Rissy2021), African institute of Biomedical science and technology(AiBST), Harare, Zimbabwerissymak@gmail.com

* [Ken Murithi](https://github.com/kenmurithi), International Center of Insect Physiology and Ecology, Nairobi, Kenya kenmurithi522@gmail.com

* [Peter Kimani](https://github.com/PMuchina), International Center of Insect Physiology and Ecology, Nairobi, Kenya kimanimuchina@gmail.com

* [UMUHOZA Diane](), Rwanda Agriculture and Animal Ressources Board, ddumuhoza@gmail.com





