# Dual RNA-Seq in a Malaria Co-infection Bacteremia


![](https://mail.google.com/mail/u/0?ui=2&ik=57afa1c84a&attid=0.1&permmsgid=msg-a:r2295766110414945034&th=17c61e032bf84e39&view=att&disp=safe&realattid=f_kuiwg6010)


## 1.	Background

**What is the current problem?**

The clinical diagnosis and distinction between malaria and bacteraemia is a challenge to clinicians due to their overlapping symptoms. This causes a potential risk of misdiagnosis and in turn inappropriate treatment therapy provision or death.

**What tools are currently used to distinguish the two?**

* Use of clinical indicators, such as bivariate and multivariate regression analyses
* Triage classification and grouping of severe infections based on Point of care rapid diagnostic test (RDT) results
* Implementation of available clinical guidelines on case-by-case management
* Risk assessment biomarker usage 
* clinical-based algorithms adoption

**What are we proposing**

We propose use of dual RNA-seq to quantify RNA transcripts of blood from patients co-infected with malaria and bacteraemia. 

**To achieve this we ought to:**
1. Use of Next generation sequencing to provide a more sensitive, specific and accurate method for the etiologic distinction of malaria and bacteraemia
2. Use differential gene expression analysis profiles to identify unique signatures between patients co-infected with malaria and bacteraemia 
3.	Determine the synergistic pathways involved

**The targets**

![image](https://user-images.githubusercontent.com/59683723/136564329-e7d964c9-e75c-4657-9ccf-56305f3b7c90.png)


##### 

## 2. Workflow of the project 

![image](https://user-images.githubusercontent.com/59683723/136552486-1f439d4a-fa42-49bf-b55f-6aedae6eddc0.png)


**Workflow Tools**


![image](https://user-images.githubusercontent.com/59683723/136560705-8d2ee026-2a2d-4120-8e85-0811e55aa016.png)



 **Source of datasets**
  - [GEO ](https://www.ncbi.nlm.nih.gov/geo/): public data repository for storing high throughput gene expression data and hybridization arrays, chips, microarrays
  - [Array-express](https://www.ebi.ac.uk/arrayexpress/):  public repository for microarray-based gene expression data
  - [ENCODE](https://www.encodeproject.org/): public research consortium that contains a list of functional elements in the human genome
  - [SRA](https://www.ncbi.nlm.nih.gov/sra) : contains raw sequence data for an experiment
  - [ENA](https://www.ebi.ac.uk/ena/browser/home) : European Nucleotide Archive, for storing raw sequencing data
  - [DDBJ-DRA](https://www.ddbj.nig.ac.jp/dra/index-e.html): stores raw sequences generated by Next Generation Sequencing (NGS)   


## 3. Team Members
* [Nzungize Lambert](https://github.com/nzungizelab), Centre for Research in Infectious Diseases (CRID/LSTM), nzulapa@gmail.com

* [Arnaud Kengne-Ouafo](https://github.com/JAKO-waccbip), Centre for Research in infectious diseases (CRID), yaound√©, Cameroon, jakengne@gmail.com

* [Rissy Makhokha](https://github.com/Rissy2021), African institute of Biomedical science and technology(AiBST), Harare, Zimbabwerissymak@gmail.com

* [Ken Murithi](https://github.com/kenmurithi), International Center of Insect Physiology and Ecology, Nairobi, Kenya kenmurithi522@gmail.com

* [Peter Kimani](https://github.com/PMuchina), International Center of Insect Physiology and Ecology, Nairobi, Kenya kimanimuchina@gmail.com

* [UMUHOZA Diane](), Rwanda Agriculture and Animal Ressources Board, ddumuhoza@gmail.com





